lib/lita/adapters/slack/message_handler.rb

```diff
@@ -171,7 +171,14 @@ def handle_message
 
           return if from_self?(user)
 
-          dispatch_message(user)
+          case data["subtype"]
+          when "message_deleted"
+            robot.trigger(:message_deleted, data)
+          when "message_changed"
+            robot.trigger(:message_changed, data)
+          else
+            dispatch_message(user)
```

Comment:
Since all of these are technically messages, perhaps it makes sense to rename this function to something like `handle_me_message`.